src/Component/Avatar/Avatar.tsx

```diff
@@ -4,16 +4,24 @@ import { AvatarCompSize, AvatarImgSize } from "../../constants";
 
 export interface ImgProps {
     imgsrc?: string;
-    order: number
+    order?: number
 };
 
 const ImgComp = styled.div<ImgProps>`
-height:${AvatarCompSize.max_height}px;
-width:${AvatarCompSize.max_width}px;
+height:${AvatarCompSize.max_height};
```

Comment:
If we are setting max_height it the css property also should be max-height
```suggestion
max-height:${AvatarCompSize.max_height};
```